test/e2e/tests/prague/upgrade-account.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { Suite } from 'mocha';
+import { Mockttp } from 'mockttp';
+import { Driver } from '../../webdriver/driver';
+import { WINDOW_TITLES, withFixtures } from '../../helpers';
+import FixtureBuilder from '../../fixture-builder';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+import ActivityListPage from '../../page-objects/pages/home/activity-list';
+import Eip7702AndSendCalls from '../../page-objects/pages/confirmations/prague/confirmation';
+import HomePage from '../../page-objects/pages/home/homepage';
+import TestDapp from '../../page-objects/pages/test-dapp';
+
+async function mockEip7702FeatureFlag(mockServer: Mockttp) {
+  return [
+    await mockServer
+      .forGet('https://client-config.api.cx.metamask.io/v1/flags')
```

Comment:
Minor, but if we do this often, we could make a generic `mockFeatureFlags` helper?

Reply:
sure, added!